GeneralizedMorphismsForCAP/gap/GeneralizedMorphismCategory.gd

```diff
@@ -284,3 +284,13 @@ DeclareOperation( "GeneralizedMorphismWithSourceAid",
 
 DeclareAttribute( "CombinedImageEmbedding",
                   IsGeneralizedMorphism );
+
+####################################
+##
+#! @Section Tools to propagate attributes
+##
+####################################
+
+DeclareGlobalVariable( "PROPAGATION_LIST_FOR_GENERALIZED_MORPHISMS" );
```

Comment:
We should document this variable to have a proper discription of what the entries of this list mean.